go.mod

```diff
@@ -3,7 +3,6 @@ module inet.af/netaddr
 go 1.12
 
 require (
-	github.com/dvyukov/go-fuzz v0.0.0-20201127111758-49e582c6c23d // indirect
```

Comment:
Hm, how do you feel about this @josharian?

Seems fine (actually even good) to me, but is there any reason you wanted this here? If you need to pin a version, do it in a shell script or Makefile that runs the fuzz?

Reply:
Every time you run go fuzz, it adds that dependency. It's long story...probably fixable, but not worth the effort. So I checked it in so that I could run go fuzz without dirtying the repo. I'd rather leave it in if possible. Or maybe there's an easy way to make go mod tidy leave it there?

Reply:
What I've seen other projects do in the same situation is adding a separate package (e.g. `tools`) importing the dependency. This will then show up in `go.mod`, i.e. for this repo something along the lines of:

```
$ mkdir tools
$ cat > tools/tools.go << EOF
// +build tools

package tools

import (
	_ "github.com/dvyukov/go-fuzz/go-fuzz-build"
)
EOF
$ go mod tidy
```

Does that sound reasonable to both of you @josharian @bradfitz?

Reply:
Fine by me.

Reply:
Pushed an additional commit adding the above.